app/controllers/child_objects_controller.rb

```diff
@@ -66,6 +66,18 @@ def destroy
     end
   end
 
+  def update_checksum
```

Comment:
Unfortunately, this needs to run on a regular worker, since the management frontend doesn't have access to the file mount.  It's probably good to run it async anyway since it may take a while.
I'd do it with an ActiveJob without a batch job.